### DISCUSSION TITLE
Tiny SD + OpenVINO + LCM

### DISCUSSION THREAD

**Discussion by jjpb44** [rupeshs/fastsdcpu#89] [Ideas]
Hello, I am trying to include a small SD in a small project.

I'm a little worried about the size of the model.

I followed the steps to run the tiny segmind sd15 model (size 240 MB [TinySD)](https://huggingface.co/segmind/tiny-sd) with OpenVINO [TinySD+OVINO](https://docs.openvino.ai/2023.0/notebooks/251-tiny-sd-image-generation-with-output.html) and the results are very good although they need the usual 20-25 samples to have good quality. One way I think this could be improved is to bake an LCM with this model before converting to openvino.
I hope this is not excessive, there is even the 480MB Small model but I see more use for the tiny model.

What can be done to achieve this workflow?

Can the same LCM from SD15 be used?
Thanks in advance

**Comment by rupeshs** [comment]
@jujpenabe Base model (SG161222/Realistic_Vision_V4.0) works with LCM-LoRA but TinySD is not compatible with LCM-LoRA sd1.5 workflow,
